lib/request-authenticator.js

```diff
@@ -38,7 +38,7 @@ class RequestAuthenticator {
       return;
     }
 
-    if (this._context.request.rawBody === undefined) {
+    if (this._context.request.body === undefined) {
```

Comment:
why? https://github.com/emartech/koa-escher-auth/pull/28/files#diff-c39afd0f80bff95808d2a019ad19938279ae8bf42ad7c1d9f4dc7a7a4c62ed74R55

Reply:
context.request.body not a defined by koa, it is also a property that is managed by koa-bodyparser. The code explicitly asserts the existence of koa-bodyparser see the error message. If you sent an empty body in POST request the rawBody will be undefined regardless of koa-bodyparser usage, however the context.request.body will be an empty object: https://www.npmjs.com/package/koa-bodyparser#usage